thinc/backends/__init__.py

```diff
@@ -113,6 +122,9 @@ def get_array_ops(arr):
 def use_ops(name: str, **kwargs):
     """Change the backend to execute on for the scope of the block."""
     current_ops = get_current_ops()
+    # avoid fallback to base NumpyOps if on big endian platform
+    if current_ops.name == "bigendian" and name == "numpy":
+        name = current_ops.name 
```

Comment:
I think we've removed all the `use_ops("numpy")` from our code, so I think I'd rather have at most a warning/error rather than having `use_ops` silently not do what you just told it to do.

Reply:
I removed the highlighted code... At best, this was my attempt to keep an explicit use_ops("numpy") specification from reverting back to numpy ops when explicitly requested from outside of thinc. It is best not to ignore an explicit request.